TROUBLESHOOTING.md

```diff
@@ -0,0 +1,98 @@
+# Troubleshooting
+
+## I can't install terraform v0.15.4+
+
+Install terraform:
+
+```
+install: https://github.com/tfutils/tfenv
+tfenv install 0.15.4
+```
+
+## Error 403: user does not have serviceusage.services.use access to the Google Cloud project
+
+Try to force auth
+```
+gcloud auth login
+```
+
+If that does not work, put you creadentials in JSON format in an environment variable as detailed in this url:
+
+```
+https://cloud.google.com/docs/authentication/production#passing_variable
+
+export GOOGLE_CREDENTIALS="/home/user/Downloads/service-account-file.json"
+```
+
+If that is not enough, modify file version.tf to add the path to the credentials file, as explained:
+```
+https://www.terraform.io/docs/language/settings/backends/gcs.html#configuration-variables
+```
+
+## Trials licenses are not being exported to the final VMs
+
+All solo.io enterprise products require a license key.  If you'd like to preset limited-term keys on the student Virtual Machines, then set the `LICENSE_KEY` and `GLOO_MESH_LICENSE_KEY` and `PORTAL_LICENSE_KEY` environment variables on your workstation before running the `terraform` command.
+
+```
+export LICENSE_KEY=VeryLongKeyString
+export GLOO_MESH_LICENSE_KEY=AnotherVeryLongKeyString
+export PORTAL_LICENSE_KEY=AnotherVeryLongKeyString
```

Comment:
We can eliminate PORTAL_LICENSE_KEY from this.  Portal licensing is now included with Edge Enterprise license.

Reply:
fixed!